### ISSUE TITLE
Receive error with -h option

### ISSUE THREAD

**Issue by k4rtik** [jas14/gcal2text#2]
Calling help shouldn't need client secret, should it? Also, it might be a good idea to update README with where and what to put in client_secret.json

```
k4rtik at PlatiniumAir in ~/Projects/gcal2text on master
$ python gcal2text.py -h
Traceback (most recent call last):
  File "gcal2text.py", line 254, in <module>
    main()
  File "gcal2text.py", line 106, in main
    credentials = get_credentials()
  File "gcal2text.py", line 46, in get_credentials
    flow = client.flow_from_clientsecrets(CLIENT_SECRET_FILE, SCOPES)
  File "/usr/local/lib/python2.7/site-packages/oauth2client/util.py", line 142, in positional_wrapper
    return wrapped(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/oauth2client/client.py", line 2210, in flow_from_clientsecrets
    cache=cache)
  File "/usr/local/lib/python2.7/site-packages/oauth2client/clientsecrets.py", line 165, in loadfile
    return _loadfile(filename)
  File "/usr/local/lib/python2.7/site-packages/oauth2client/clientsecrets.py", line 125, in _loadfile
    raise InvalidClientSecretsError('File not found: "%s"' % filename)
oauth2client.clientsecrets.InvalidClientSecretsError: File not found: "client_secret.json"
```

**Comment by jas14** [comment]
There were two options here:
1. require each user to create their own Google App, set up the GCal API, get the client ID + secret, and pass those in, or
2. add my client secrets.

I opted for the second because:
- the first effectively cancels out the convenience that the script is supposed to provide in the first place
- according to Google, installed apps aren't really expected to keep client secrets "secret" anyway

I realize that a malicious user could write a different installed app and masquerade as gcal2text if they so desired. I assume anyone using this script has either looked at the source or blindly (but correctly) believes any script using the gcal2text keys isn't doing something nasty. If it's the former, that person would presumably want to look at the source of other scripts they run that might try to masquerade as mine. If it's the latter, then that's on the user.

I also realize that an attacker could issue millions of requests and exhaust my quota. That would be  irritating, but it wouldn't really cause any damage.

Anyway, TLDR fixed this issue. Thanks for filing @k4rtik !

**Comment by k4rtik** [comment]
:+1:
